src/DIRAC/WorkloadManagementSystem/scripts/dirac-admin-sync-pilot.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python
```

Comment:
Can this be factorised somewhere given part of it is almost idential to `PilotsSyncAgent.execute`?

Reply:
I thought about it, but there are still some small differences that would make the factorisation more a nuisance than a useful thing.